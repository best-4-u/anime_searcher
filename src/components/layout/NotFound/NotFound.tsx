import { FC } from "react";

import styles from "./NotFound.module.scss";

const NotFound: FC = () => {
  return (
    <div className={styles.not_found}>
      <h1> 404 Not found </h1>
    </div>
  );
};

export default NotFound;
