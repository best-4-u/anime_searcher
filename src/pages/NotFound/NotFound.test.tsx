import { create } from "react-test-renderer";

import NotFound from "./NotFound";

describe("Not found", () => {
  it("renders correctly snapshot", () => {
    const tree = create(<NotFound />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
