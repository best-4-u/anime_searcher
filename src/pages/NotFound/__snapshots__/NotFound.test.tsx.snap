// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Not found renders correctly snapshot 1`] = `
<div
  className="not_found"
>
  <h1>
     404 Not found 
  </h1>
</div>
`;
